core/themes/basis/css/layout.css

```diff
@@ -5,7 +5,6 @@
 
 .l-header {
   position: relative;
-  margin: 0 0 2rem;
```

Comment:
I think we do this on home only, not everywhere.

Another option is we could put `margin-top: -2rem;` on the top region. So if it's below header it doesn't get the distance from the top.